File: ch10/listargs.c
#include<stdio.h>

int main(int argc, char const *argv[])
{
    printf("Number of args :%d,Args are:\n",argc);

    for (size_t i = 0; i < argc; i++)
    {
        printf("args[%d] %s\n",i,argv[i]);
    }
    fprintf(stderr,"This message is sent to stderr");
    return 0;
}

File: ch10/ch10_src/src/bin/listargs.rs
use std::env::args;

fn main(){
    let argv = args();
    println!("Number of args :{},Args are:",argv.len());
    for (i,arg) in argv.enumerate() {
        println!("args[{}] {}",i,arg);
    }
    eprint!("This message is sent to stderr");
}